### PR TITLE
Task/purge error table

### PR DESCRIPTION
**Pull request by AlvaroVega**
- [x] Tested in MySQL and Postgis
- [x] update doc
issue https://github.com/telefonicaid/fiware-cygnus/issues/1929

### CODE REVIEW

File: docker/cygnus-common/Dockerfile [hunk 1]
```diff
@@ -36,7 +36,7 @@ ENV CYGNUS_API_PORT "8081"
 # PR https://github.com/telefonicaid/fiware-cygnus/pull/1801 should be also done in this Dockerfile
 # NOTE: Configure correctly GIT_URL_CYGNUS and GIT_REV_CYGNUS for each git branch/fork used
 ENV GIT_URL_CYGNUS "https://github.com/telefonicaid/fiware-cygnus.git"
-ENV GIT_REV_CYGNUS "master"
+ENV GIT_REV_CYGNUS "task/purge_error_table"
```
Comment (fgalan) [inline docker/cygnus-common/Dockerfile:39]:
```suggestion
ENV GIT_REV_CYGNUS "master"
```

(Seems to be a local change not changed before pushing the commit to github)

File: doc/cygnus-common/installation_and_administration_guide/error_persistance.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ If there is an exception when trying to persist data into storage. Cygnus is cap
 This only applies to SQL type backends PostgreSQL & MySQL. Notice that Postgis uses PostgreSQL backend so this function extends to Postgis as well as PostgreSQL and MySQL.
 
 This feature is enabled by default but could be disabled by setting sink option `persist_errors` to false.
+The among of errors which are persisted is the latest 100 by default, but couldd be changed by setting sink option `max_latest_errors`.
```
Comment (fgalan) [inline doc/cygnus-common/installation_and_administration_guide/error_persistance.md:8]:
```suggestion
The among of errors which are persisted is the latest 100 by default, but could be changed by setting sink option `max_latest_errors`.
```

File: CHANGES_NEXT_RELEASE [hunk 3]
```diff
@@ -1,3 +1,4 @@
-[cygnus-common][SQLBackendImpl] Add configuration option to persist errors in SQL sinks (#1928)
+[cygnus-common][SQLBackendImpl] Purge automatically persistence error logs table, keeping latest errors (#1929)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
[cygnus-common][SQLBackendImpl] Purge automatically persistence error logs table, keeping latest errors (max_latest_errors setting) (#1929)
```

**Review by fgalan — APPROVED** [review]
LGTM

But maybe some other (@IvanHdzC or @manucarrace ) should provide feedback before merging.

**Review by IvanHdzC — APPROVED** [review]
LGTM

**Pull request merged**